pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "colorama>=0.4.6",
     "jf-ingest==0.0.203",
 ]
-requires-python = ">=3.10"
+requires-python = "~=3.10.15"
```

Comment:
This was the only way I could get `pdm` to figure out using python `3.11` was unacceptable, for whatever reason.

Reply:
Probably wanted `>=3.10,<3.11`